src/TestCase.php

```diff
@@ -41,10 +34,6 @@ public function setUp()
         if (! $this->app) {
             $this->refreshApplication();
         }
-
-        if (! $this->factory) {
-            $this->factory = $this->app->make(Factory::class);
```

Comment:
And yet oddly this PR still break because it doesn't include "faker" as a dep.

https://travis-ci.org/orchestral/testbench/jobs/70601630